app/code/Magento/Tax/Model/TaxRateCollection.php

```diff
@@ -70,6 +70,21 @@ public function loadData($printQuery = false, $logQuery = false)
         return $this;
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
I believe it worth specifying the phpdoc
```suggestion
    /**
     * @param string $colName
     * @return array
     */
```

Reply:
Agree with internal coding standard https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#inheritdoc

But in my opinion docblock contained only params don't provide any really useful information. It should contain some information what method do

Reply:
Done